.lintstagedrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
configuration file for the staged files that will run a linter script in the actual commit